makefile/.konveyor/ansible/.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
```

Comment:
Is this file required here? I mean ansible do have different then isort/flake8 targets?